yb-voyager/cmd/exportData.go

```diff
@@ -168,6 +180,106 @@ func exportDataOffline() bool {
 	return true
 }
 
+func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName) error {
+	absExportDir, err := filepath.Abs(exportDir)
+	if err != nil {
+		return fmt.Errorf("failed to get absolute path for export dir: %v", err)
+	}
+
+	var snapshotMode string
+	if liveMigration {
+		snapshotMode = "initial"
+	} else if useDebezium {
+		snapshotMode = "initial_only"
+	}
+
+	config := &dbzm.Config{
+		SourceDBType: source.DBType,
+		ExportDir:    absExportDir,
+		Host:         source.Host,
+		Port:         source.Port,
+		Username:     source.User,
+		Password:     source.Password,
+
+		DatabaseName: source.DBName,
+		SchemaNames:  source.Schema,
+		TableList:    sqlname.GetMinQuotedTargetTableList(tableList),
```

Comment:
The word `"Target"` in the method name is confusing--we're dealing with source names here.

The method is not going to be much reusable. I would suggest to simply inline the code at line 195.

And, I think in the implementation of the `GetMinQuotedTargetList()` you should use `tableName.Qualified.MinQuoted`. That will remove the need of adding schema prefix in the `getConfig()` method.